pkgs/applications/editors/emacs/build-support/generic.nix

```diff
@@ -26,7 +26,6 @@ in
 { pname
```

Comment:
definition `version` is not used

Reply:
Hey @inclyc, what about ignoring errors not relevant to the PR, such as existing ones?